Sources/Node/Core/NodeRepresentable.swift

```diff
@@ -0,0 +1,30 @@
+public protocol NodeRepresentable {
+    /// Able to be represented as a Node
+    ///
+    /// - throws: if convertible can not create a Node
+    /// - returns: a node if possible
+    func makeNode(in context: Context?) throws -> Node
```

Comment:
Why is there `in context: Context` still? Shouldn't you access the context from `node.context`?